app/scripts/lib/createMetaRPCHandler.js

```diff
@@ -15,24 +15,35 @@ const createMetaRPCHandler = (api, outStream) => {
       });
       return;
     }
-    api[data.method](...data.params, (err, result) => {
-      if (outStream._writableState.ended) {
-        return;
-      }
-      if (err) {
-        outStream.write({
-          jsonrpc: '2.0',
-          error: serializeError(err, { shouldIncludeStack: true }),
-          id: data.id,
-        });
-      } else {
-        outStream.write({
-          jsonrpc: '2.0',
-          result,
-          id: data.id,
-        });
+
+    let result;
+    let error;
+    try {
+      result = await api[data.method](...data.params);
+    } catch (err) {
+      error = err;
+    }
+
+    if (outStream._writableState.ended) {
+      if (error) {
+        console.log(error);
```

Comment:
Oops, this was meant to be `console.error`.

```suggestion
        console.error(error);
```